Add new field sessionId into Session (#548)
cognitedata/cdp-spark-datasource: CHANGELOG.md

# 1.4.47

## Fixes

> * Add sessionId into Session as it is requires when refreshing the session

Reviewer: ```suggestion
* Add the new field `sessionId` to session as it is required when refreshing the session.
```
